aircraft/cost.py

```diff
@@ -0,0 +1,112 @@
+"""Aircraft cost models"""
+from gpkit.shortcuts import Var, Model
+import gpkit
+gpkit.disable_units()
```

Comment:
Likewise. Should we just convert things like `$/hr` to `1/hr` for now, i.e. assume dollars are dimensionless?